nixos/doc/manual/development/running-nixos-tests-interactively.section.md

```diff
@@ -5,7 +5,7 @@ when developing or debugging a test:
 
 ```ShellSession
 $ nix-build nixos/tests/login.nix -A driverInteractive
-$ ./result/bin/nixos-test-driver
+$ ./result/bin/nixos-test-driver --interactive
```

Comment:
What's the point of having a `driverInteractive` attribute if you still have to pass a flag to make it interactive? At the very least, the `driverInteractive` attribute should add the new `--interactive` flag.

Reply:
As of https://github.com/NixOS/nixpkgs/pull/156697, this looks like it's been resolved 🎉